lms/static/js/edxnotes/views/notes_page.js

```diff
@@ -34,20 +36,23 @@ define([
                 scrollToTag: scrollToTag
             });
 
-            this.courseStructureView = new CourseStructureView({
-                el: this.el,
-                collection: this.collection,
-                tabsCollection: this.tabsCollection,
-                scrollToTag: scrollToTag
-            });
+            if (!_.contains(this.options.disabledTabs, 'course_structure')) {
+                this.courseStructureView = new CourseStructureView({
+                    el: this.el,
+                    collection: this.collection,
+                    tabsCollection: this.tabsCollection,
+                    scrollToTag: scrollToTag
+                });
 
-            // Add the Tags model after the Course Structure model.
-            this.tabsCollection.push(tagsModel);
+                // Add the Tags model after the Course Structure model.
+                this.tabsCollection.push(tagsModel);
```

Comment:
Pushing this into the collection should not depend on whehter `course_structure` tab is enabled or not. It should be a separate `if`.